app/assets/stylesheets/pageflow/editor/page_links.css.scss

```diff
@@ -12,6 +12,14 @@ div.page_links {
     @include icon-only-button(#aaa);
     @include pencil-icon;
     position: absolute;
+    right: 40px;
+    top: 4px;
+  }
+
+  a.remove {
+    @include icon-only-button(#aaa);
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.